tests/unit-tests/validateRunnerVersion-test.js

```diff
@@ -0,0 +1,40 @@
+const { expect } = require('chai');
+
+const packageJson = require('../../package.json');
+const runnerValidator = require('../../src/common/validateRunnerVersion');
+
+describe('validateRunnerVersion', function() {
```

Comment:
- [ ] add test with only MAJOR.MINOR tagged image
- [ ] add test without any version number in the tag (`zooz/predator:test`)